apps/remix-ide/src/app/files/remixd-handle.js

```diff
@@ -82,6 +84,7 @@ export class RemixdHandle extends WebsocketPlugin {
           }
         }, 3000)
         this.locahostProvider.init(_ => this.fileSystemExplorer.ensureRoot())
+        this.appManager.ensureActivated('git')
```

Comment:
This has been removed in #651. Why not running `this.call('manager', 'activatePlugin', 'git')` ?
Or even let the engine lazily activate the git plugin on first call